lib/mongoid/embedded_errors.rb

```diff
@@ -36,7 +38,7 @@ def errors_with_embedded_errors
                     "#{name}.#{k}"
                   end.to_sym
             errs.delete(key)
-            errs.add key, v if v.present?
+            errs.add key, v
```

Comment:
I thought this `if v.present?` check is necessary, why was it removed?

Reply:
its not needed. we are skipping if any errors.any? on line 26.

Reply:
I think this still might be needed in some edge cases where key is added to the errors array, but value is an empty string or something like that. Let's keep it for now, just to be safe.